components/EditSidebar/Checklist.js

```diff
@@ -100,19 +108,31 @@ class Checklist extends Component {
                 >
                   {t(`checklist/labels/${name}`, undefined, name)}
                   {!!author && <span {...styles.approvedBy}>
-                    {cleanName(author.name)}
-                  </span>}
-                  {!!commit && <span {...styles.approvedBy}>
-                    {timeFormat(
-                      new Date(commit.date)
-                    )}
-                    {' '}
-                    {(
-                      author.name !== commit.author.name &&
-                      cleanName(commit.author.name)
-                    )}
+                    {cleanName(author.name)} {t('checklist/approvedFor')}
```

Comment:
Maybe use a translate variable here—help to understand the context in the translations file
`t('checklist/approvedFor', {name: cleanName(author.name)})`

`{name} für`

Reply:
Done.